<launch>
    <group ns="pandora_vision_hazmat">
      
      <param name="camera_name" value="kinect" />
    
      <node name="hazmat_node" type="hazmat_node" pkg="pandora_vision_hazmat" respawn="False" output="screen" />
      <!-- Add in upper line to enable debugging: launch-prefix="kdbg" /-->
      
      <!-- **************** /-->
      <!-- Debug Mode params /-->
      <!-- **************** /-->
  
        <!-- Enable Hazmat Debug /-->
        <param name="debugHazmat" type="bool" value="True"/>
  
    </group>
</launch>
